spikes/stl_cxxsupport/main.cpp

```diff
@@ -94,5 +95,20 @@ auto main() -> int
 	auto back = span0.back();
 	log_info("span0.back() = %i", back);
 
+	log_info("Test std::range features");
+
+	constexpr auto range = [](const auto a, const auto b) {
+		auto x = std::ranges::views::iota(a) | std::ranges::views::take(b - a + 1);
+		return std::ranges::views::all(x);
```

Comment:
Why don't use directly 

```suggestion
		return std::ranges::views::iota(a, b);
```

Reply:
**Note Important.** Range go from a included to b excluded in that suggestion (  `[a, b[`  )

Reply:
https://gcc.godbolt.org/z/7oc8bhqde